### DISCUSSION TITLE
Não consigo adiconar icone

### DISCUSSION THREAD

**Discussion by Thomaz-Castro** [marcelotduarte/cx_Freeze#1500] [Q&A]
Crio meu setup tudo certinho, igual videos no youtube, o programa funciona 100%, exeto o icone que não é adicionado, o que eu faço?
![image](https://user-images.githubusercontent.com/103527232/169173104-ddc06ec3-a3c6-406d-9c18-9b136ee7a924.png)
![image](https://user-images.githubusercontent.com/103527232/169173362-246c541e-e73b-460b-b046-9fc15dfa1759.png)

**Comment by marcelotduarte** [comment]
@Thomaz-Castro Converti para Discussion.
O código quando colar, cole texto, pois se precisar reescrever parte fica mais fácil.
O icone.ico está aparecendo na pasta pq vc adicionou com include_files.
Mas o icone no progama deve estar falhando por ser incompativel (por exemplo, se for um png q apenas teve a extensão modificada para ico).
Veja o relatório produzido ao rodar o comando e veja se não tem WARNING relativo a isso.
